tx.go

```diff
@@ -185,6 +186,9 @@ func (tx *Tx) Commit() error {
 
 	// If the high water mark has moved up then attempt to grow the database.
 	if tx.meta.Pgid() > opgid {
+		_ = errors.New("")
+		// gofail: var lackOfDiskSpace string
+		// return errors.New(lackOfDiskSpace)
```

Comment:
It changes the behaviour. When grow() fails, it should execute tx.rollback(). 
```suggestion
		// gofail: var lackOfDiskSpace string
		// tx.rollback()
		// return errors.New(lackOfDiskSpace)
```

Reply:
Good to know. Fixed!